docs/docs/Swapchain.html

```diff
@@ -64,31 +64,34 @@ <h3>Externs</h3>
 </details>
 <details>
     <summary>
-        <pre class='prettyprint'>extern std::vector&lt;VkImage&gt; swapchain_images;</pre>
+        <pre class='prettyprint'>extern VkImage* swapchainImages;</pre>
     </summary>
     <!--COMMENT-->
 </details>
 <details>
     <summary>
-        <pre class='prettyprint'>extern VkSurfaceCapabilitiesKHR surface_capabilities;</pre>
+        <pre class='prettyprint'>extern VkSurfaceCapabilitiesKHR surfaceCapabilities;</pre>
     </summary>
-    <!--COMMENT-->
+    Contains the supported image usage falgs
```

Comment:
Is it again a typing error?

Reply:
These are auto generated files via our docs system this will change automatically if I recompile the docs!